backend/cfg/vectorize.ml

```diff
@@ -2,13 +2,145 @@
 
 module DLL = Flambda_backend_utils.Doubly_linked_list
 
-let dump ppf cfg_with_layout ~msg =
+module Instruction = struct
```

Comment:
what do you mean regalloc?

Reply:
"regalloc" stands for register allocation.

Reply:
Indeed it would be nice to have something
along these lines used in more place. In
particular I would like to make the type of
instruction identifier abstract.